### DISCUSSION TITLE
Authelia - Authentication and User Management

### DISCUSSION THREAD

**Discussion by x0rb** [coollabsio/coolify#4689] [Service Template Requests]
I noticed many apps require login to work. 

Outline (getoutline.com) doesn't work without SSO/Authentik/Authelia to manage authentication.

Is there a reason why Authelia isn't offered as a one-click install by Coolify?

I understand Authentik is better but Authelia is easier for services that only I will use.

It's not a big deal but thought I'd ask what the reason was behind authentik > authelia.

**Comment by platon-ivanov** [comment]
The apps are added by community members, and there is an ongoing PR for Authelia: https://github.com/coollabsio/coolify/pull/5947.
